projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/intro.json

```
@@ -1,11 +1,11 @@
 {
-  "title": "Melvin's Guide to Gadgets",
+  "title": "Melvin的工匠装置宝典",
   "text": [
     {
-      "text": "Hey future or current tinker. My name is Melvin. I fell in a slimepool once, and since then I have never stopped experimenting with slime. While I am not a master tinker yet, I am hoping some of my discoveries will one day get me accepted into their ranks."
+      "text": "嘿，未来或是现在的工匠。我叫Melvin。有一次我掉进了黏液池里，从那以后我就一直在尝试利用黏液制造一些奇特的小玩意。虽然我还并不是一个工匠大师，但我希望有一天因为我的一些发现能让我加入工匠大师的行列。"
```

```suggestion
      "text": "嘿，工匠以及准工匠们。我是Melvin。有一次我掉进了黏液池里，从那以后我就一直在尝试利用黏液制造一些奇特的小玩意。虽然我还并不是一个工匠大师，但我希望有一天因为我的一些发现能让我加入工匠大师的行列。"
```
第一句原文没有复数，但我觉得复数翻译过来更舒服。